x/withdraw/ibc_middleware.go

```diff
@@ -0,0 +1,156 @@
+package withdraw
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	porttypes "github.com/cosmos/ibc-go/v3/modules/core/05-port/types"
+	"github.com/cosmos/ibc-go/v3/modules/core/exported"
+
+	"github.com/tharsis/evmos/v2/x/withdraw/keeper"
+)
+
+var _ porttypes.Middleware = &IBCMiddleware{}
+
+// IBCMiddleware implements the ICS26 callbacks for the transfer middleware given
+// the claim keeper and the underlying application.
```

Comment:
```suggestion
// the withdraw keeper and the underlying application.
```

Isn't the withdraw keeper being passed into `NewIBCMiddleware`